pertpy/tools/_mixscape.py

```diff
@@ -194,7 +194,7 @@ def mixscape(
         if layer is not None:
             X = adata_comp.layers[layer]
         else:
-            X = adata_comp.X
+            X = adata_comp.layers["X_pert"]
```

Comment:
We could wrap this with a try except and if the layer cannot be found we suggest to the user to calculate the perturbation signature first. Think this is what you wanted with your warning earlier?

Reply:
exactly

Reply:
Could you have an attempt, please?
Use `print("[bold yellow] BLABLA")` for warnings

Reply:
I used try. What do you think?